tailwind.config.ts

```diff
@@ -1,20 +1,210 @@
+  safelist: [
+    {
+      pattern: /bg-[a-z0-9-]*/,
+    },
   ],
   theme: {
     extend: {
       backgroundImage: {
-        'gradient-radial': 'radial-gradient(var(--tw-gradient-stops))',
-        'gradient-conic':
-          'conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))',
+        "gradient-radial": "radial-gradient(var(--tw-gradient-stops))",
+        "gradient-conic":
+          "conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
+      },
+    },
+    boxShadow: {
+      0: "0px 0px 0px 0px var(--effect-shadow-e0)",
+      1: "0px 1px 3px -1px var(--effect-shadow-e3)",
+      2: "0px 3px 4px -2px var(--effect-shadow-e3)",
+      3: "0px 8px 24px -4px var(--effect-shadow-e2)",
+      4: "0px 18px 32px -8px var(--effect-shadow-e2)",
+      5: "0px 28px 24px -8px var(--effect-shadow-e1)",
+      6: "0px 40px 48px -8px var(--effect-shadow-e1)",
+    },
+    fontSize: {
+      "2xs": ["10px", "16px"],
+      xs: ["11px", "16px"],
+      sm: ["12px", "16px"],
+      base: ["14px", "24px"],
+      md: ["16px", "24px"],
+      lg: ["18px", "28px"],
+      xl: ["20px", "32px"],
+      "2xl": ["26px", "40px"],
+      "3xl": ["32px", "40px"],
+    },
+    borderRadius: {
+      0: "0px",
+      4: "4px",
+      6: "6px",
+      8: "8px",
+      12: "12px",
+      16: "16px",
+      20: "20px",
+      32: "32px",
+      48: "48px",
+      64: "64px",
+      100: "100px",
+    },
+    colors: {
+      outline: {
+        none: "var(--outline-none)",
+        white: "var(--outline-white)",
+        black: "var(--outline-black)",
+        "base-em": "var(--outline-base-em)",
+        "low-em": "var(--outline-low-em)",
+        "med-em": "var(--outline-med-em)",
+        "high-em": "var(--outline-high-em)",
+        primary: {
+          "base-em": "var(--outline-primary-base-em)",
+          "low-em": "var(--outline-primary-low-em)",
+          "med-em": "var(--outline-primary-med-em)",
+          "high-em": "var(--outline-primary-high-em)",
+        },
+        success: {
+          "base-em": "var(--outline-success-base-em)",
+          "low-em": "var(--outline-success-low-em)",
+          "med-em": "var(--outline-success-med-em)",
+          "high-em": "var(--outline-success-high-em)",
+        },
+        danger: {
+          "base-em": "var(--outline-danger-base-em)",
+          "low-em": "var(--outline-danger-low-em)",
+          "med-em": "var(--outline-danger-med-em)",
+          "high-em": "var(--outline-danger-high-em)",
+        },
+        info: {
+          "base-em": "var(--outline-info-base-em)",
+          "low-em": "var(--outline-info-low-em)",
+          "med-em": "var(--outline-info-med-em)",
+          "high-em": "var(--outline-info-high-em)",
+        },
+        warning: {
+          "base-em": "var(--outline-warning-base-em)",
+          "low-em": "var(--outline-warning-low-em)",
+          "med-em": "var(--outline-warning-med-em)",
+          "high-em": "var(--outline-warning-high-em)",
```

Comment:
I can see an opportunity to simplify this. With the same name convention a function might make this super simple. 

Just an idea, it's okay as it is.

Reply:
(maybe it's too early)

Reply:
Will give it a think. This can be interesting but might be too soon.